README.md

```diff
@@ -15,6 +15,12 @@ The service is compatible with Heroku, and can be deployed by clicking the butto
 
 [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/manywho/service-dummy/tree/develop)
 
+If executed without any arguments, the service starts in V1 mode over http. To start in V2 mode with https client authentication, supply the following arguments;
+
+`service-dummy.jar true /path/to/server_keystore.jks /path/to/server_truststore.jks` 
```

Comment:
Perhaps it would be a good idea to explain what are these arguments? What is the difference between server_keystore.jks and server_truststore.jks?

Reply:
Done